### PR TITLE
add email widget

### PR DESCRIPTION
**Pull request by zepatrik**
As the type of an email field is `string` with format `email` it was not rendering an email field before. You could abstract this either with a factory function or a component that takes the format into consideration and is registered as `TextWidget` and `EmailWidget`. If you prefer another solution I'll happily adapt this PR

### CODE REVIEW

**Comment by agustin107** [comment]
Thanks for your contribution!

Can you update the PR deleting `package-lock.json` and use `yarn` instead `npm`?

**Comment by zepatrik** [comment]
So do you want a yarn lock file or none?

**Comment by agustin107** [comment]
Yep! We have versioned the `yarn.lock` file.. so git will detect as change and not as a new file.

**Comment by zepatrik** [comment]
ok done @agustin107

File: src/EmailWidget/EmailWidget.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      //error={!!rawErrors}
+      required={required}
+    >
+      <TextField
+        id={id}
+        label={label || schema.title}
+        autoFocus={autofocus}
+        required={required}
+        disabled={disabled || readonly}
+        name={name}
```
Comment (agustin107) [inline src/EmailWidget/EmailWidget.tsx]:
```suggestion
```
Remove it as we removed on #81

**Pull request merged**